addons/project_todo/static/tests/mock_server/mock_models/mail_activity_todo_create.js

```diff
@@ -0,0 +1,11 @@
+import { fields, models } from "@web/../tests/web_test_helpers";
+
+
+export class MailActivityTodoCreate extends models.Model {
```

Comment:
Why do we keep old tests ? 

Could you remove them please ?

Reply:
ld tests have been removed as per your suggestion. 
Thank you!

Reply:
> Why do we keep old tests ?
> 
> Could you remove them please ?

Done. After looking at other modules like Planning, we had moved it to the legacy folder instead of removing it.

Reply:
Yep indeed, but if the conversion is correctly made then we can now directly remove those tests.

The planning tests could also be used in inherited modules for instance.